terraform/app/modules/paas/variables.tf

```diff
@@ -96,6 +96,15 @@ variable "enable_logit" {}
 
 variable "aks_web_app_memory" {}
 
+variable "enable_dfe_analytics_federated_auth" {
+  description = "Create the resources in Google cloud for federated authentication and enable in application"
+  default     = false
```

Comment:
Can we remove the variable defaults at the module level? Defaults are already set at the root level

Reply:
Removed.